packages/builder/src/components/common/LinkedRowSelector.svelte

```diff
@@ -15,14 +15,26 @@
   let rows = []
   let linkedIds = []
 
-  $: linkedIds = (Array.isArray(linkedRows) ? linkedRows : [])?.map(
-    row => row?._id || row
-  )
+  $: fieldValue = getFieldValue(linkedData)
   $: label = label || capitalise(schema.name)
   $: linkedTableId = linkedTableId || schema.tableId
   $: linkedTable = $tables.list.find(table => table._id === linkedTableId)
   $: fetchRows(linkedTableId)
 
+  const getFieldValue = val => {
```

Comment:
This consumes `schema` but it's not a parameter - so we might want to update this so the reactive statement also passes it in.

Reply:
Great catch!